husky-fhir/husky-fhir-emed-ch/husky-fhir-emed-ch-epr/src/main/java/org/projecthusky/fhir/emed/ch/epr/resource/ChEmedEprMedicationRequest.java

```diff
@@ -417,7 +421,7 @@ public void copyValues(final MedicationRequest dst) {
     @Override
     @ExpectsValidResource
     public Author resolveMedicalAuthor() {
-        return resolveRequester();
+        return new Author(this.resolveRequester());
```

Comment:
```suggestion
        return new Author(this.resolveRequester(), this.resolveAuthoredOn());
```

Reply:
Unless you object for some reason, I'd prefer to keep the authorship timestamp information on the reply of this method.

Reply:
I agree, it was not my intent to remove it. I saw that when working on #154